p4app-exercises/p4runtime.p4app/solution/mycontroller.py

```diff
@@ -68,14 +75,18 @@ def writeTunnelRules(p4info_helper, ingress_sw, egress_sw, tunnel_id,
 
     # TODO build the transit rule
     # TODO install the transit rule on the ingress switch
+    if direction == "clockwise":
```

Comment:
honestly I would just add ingress_sw_port_number_to_egress_sw as a function parameter here instead of direction

Reply:
I have made another commit changing thing as you suggest, calling the parameter 'port' instead, but can make it longer if you prefer.

Reply:
maybe ingress_sw_port then as a middle ground, since this is the port number for the ingress_sw and not the egress_sw?

Reply:
Sold!